deploy/operator.yaml

```diff
@@ -50,30 +50,32 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
-            - name: NODE_NETWORK_STATE_REFRESH_INTERVAL
-              valueFrom:
-                configMapKeyRef:
-                  name: nmstate-config
-                  key: node_network_state_refresh_interval
-            - name: INTERFACES_FILTER
-              valueFrom:
-                configMapKeyRef:
-                  name: nmstate-config
-                  key: interfaces_filter
             - name: ENABLE_PROFILER
               value: "False"
             - name: PROFILER_PORT
               value: "6060"
+            - name: CONFIG_PATH
+              value: "/etc/config/nmstate.yaml"
           volumeMounts:
           - name: dbus-socket
             mountPath: /run/dbus/system_bus_socket
+          - name: config-volume
+            mountPath: /etc/config
           securityContext:
             privileged: true
       volumes:
       - name: dbus-socket
         hostPath:
           path: /run/dbus/system_bus_socket
           type: Socket
+      - name: config-volume
+        configMap:
+          # Provide the name of the ConfigMap containing the file
+          # configmap must be created from file it's full path must
```

Comment:
How is the configmap created is not important. The format is. Could you please provide an example of the body instead?

With something like:

```
data:
  nmstate.yaml: |
    key: value
    key: value

Reply:
It is important. If it is not created from file. It will not work.

Reply:
I'm pretty sure user can just create a config map which would be a prepared manifest with populated nmstate.yaml key. I haven't tried it yet, but I believe that --from-file is just a helper in kubectl to do exactly that.

Reply:
Could you dump ConfigMap with -o yaml for me?

Reply:
maybe you are right i will check.

Reply:
Checked your right